apify-docs-theme/src/theme/custom.css

```diff
@@ -346,14 +594,13 @@ aside button[class*="collapseSidebarButton"] svg {
 }
 
 nav.navbar .dropdown__menu {
-    min-width: 6rem;
+    min-width: 96px;
```

Comment:
The padding is too small now. I'd do 12/16px 
![image](https://github.com/apify/apify-docs/assets/77627332/67b0672e-e0e2-40b3-9c2b-fa7dede7e0c2)